src/tests/work-item/work-item-list/assigne.spec.js

```diff
@@ -0,0 +1,149 @@
+/**
```

Comment:
Just wondering - why is the file named "assigne"? Shouldn't this be "assign" or "assigned"?